x-pack/plugins/cases/common/api/cases/case.ts

```diff
@@ -206,31 +206,18 @@ export const CasePostRequestRt = rt.intersection([
 ]);
 
 const CasesFindRequestSearchFieldsRt = rt.keyof({
-  'closed_by.username': null,
-  'closed_by.full_name': null,
-  'closed_by.email': null,
-  'closed_by.profile_uid': null,
-  'created_by.username': null,
-  'created_by.full_name': null,
-  'created_by.email': null,
-  'created_by.profile_uid': null,
   description: null,
-  'connector.name': null,
-  'connector.type': null,
-  'external_service.pushed_by.username': null,
-  'external_service.pushed_by.full_name': null,
-  'external_service.pushed_by.email': null,
-  'external_service.pushed_by.profile_uid': null,
-  'external_service.connector_name': null,
-  'external_service.external_id': null,
-  'external_service.external_title': null,
-  'external_service.external_url': null,
   title: null,
-  'title.keyword': null,
```

Comment:
Shall I keep the `title.keyword` in searchFields @cnasikas ?

Reply:
I believe we should not keep it. The UI and the consumers should not be aware of the internals of ES. We do the mapping ourselves.